### PR TITLE
Added properties

### PR DESCRIPTION
**Pull request by dperezcabrera**

### CODE REVIEW

File: src/main/java/org/jboss/maven/plugins/bombuilder/BuildBomMojo.java [hunk 1]
```diff
@@ -41,43 +41,43 @@
...
-    @Parameter( required = true )
+    @Parameter( defaultValue = "artifact-bom", property = "bomArtifactId" )
     private String bomArtifactId;
 
     /**
      * BOM version
      */
-    @Parameter( required = true )
+    @Parameter( defaultValue = "1.0.0-SNAPSHOT", property = "bomVersion" )
     private String bomVersion;
```
Comment (dosoudil) [inline src/main/java/org/jboss/maven/plugins/bombuilder/BuildBomMojo.java:57]: Why are these mandatory parameters changed to optional with default value?

File: pom.xml [hunk 2]
```diff
@@ -22,12 +22,12 @@
...
     <artifactId>jboss-parent</artifactId>
-    <version>19</version>
+    <version>36</version>
   </parent>
 
-  <groupId>org.jboss.maven.plugins</groupId>
+  <groupId>com.github.dperezcabrera</groupId>
   <artifactId>bom-builder-maven-plugin</artifactId>
-  <version>1.2.1-SNAPSHOT</version>
+  <version>1.2.1</version>
```
Comment (dosoudil) [inline pom.xml:30]: Feature commits shouldn't include release commit.

File: pom.xml [hunk 3]
```diff
@@ -22,12 +22,12 @@
   <parent>
     <groupId>org.jboss</groupId>
     <artifactId>jboss-parent</artifactId>
-    <version>19</version>
+    <version>36</version>
   </parent>
 
-  <groupId>org.jboss.maven.plugins</groupId>
+  <groupId>com.github.dperezcabrera</groupId>
```
Comment (dosoudil) [inline pom.xml:28]: Why does the groupId is changed from JBoss to personal one?

Comment (dperezcabrera) [reply pom.xml:28]: I'm  so sorry, I just wanted to send the first commit as PR. 
Later I just did some other tests..

**Pull request closed without merging**